projects/ui-policy-studio-angular/src/lib/components/flow-details-phase/gio-ps-flow-details-phase.component.html

```diff
@@ -24,76 +24,90 @@
     {{ description }}
   </span>
 
-  <span *ngIf="isDisabled" class="header__infoIcon" matTooltip="Policies might not be evaluated" matTooltipPosition="before">
+  <span
+    *ngIf="unsupportedForThisStartEndConnector && !disabledNotYetAvailable"
+    class="header__infoIcon"
+    matTooltip="Policies might not be evaluated"
+    matTooltipPosition="before"
+  >
     <mat-icon svgIcon="gio:info"></mat-icon>
   </span>
 </div>
 
-<div *ngIf="!isDisabled; else disabledPhaseContent" class="wrapper">
-  <div class="content">
-    @for (stepVM of stepsVM; track stepVM._id; let isFirst = $first) {
-      <div class="content__step">
-        <!-- Connector -->
-        <ng-container *ngIf="stepVM.type === 'connectors'">
-          <div class="content__step__connector">
-            <span *ngFor="let connector of stepVM.connectors" class="gio-badge-white content__step__connector__badge">
-              <mat-icon *ngIf="connector.icon" [svgIcon]="connector.icon" class="content__step__connector__badge__icon"></mat-icon>
-              {{ connector.name }}
-            </span>
-          </div>
+@if (disabledNotYetAvailable) {
+  <div class="disabledContent">
+    <gio-banner-info>
+      Coming soon
+      <span gioBannerBody> This feature is not yet available in the current version of the API Studio. Stay tuned for updates. </span>
+    </gio-banner-info>
+  </div>
+} @else {
+  <div *ngIf="!unsupportedForThisStartEndConnector; else unsupportedForThisStartEndConnectorContent" class="wrapper">
```

Comment:
What do you think about converting the `*ngIf` to control flows like what has been added above?

Reply:
Ha I left it like that because there is the else with a template before
but I can convert to if else, else